interface/src/app/gene/network/network.tpl.html

```diff
@@ -17,19 +17,17 @@ <h3 class="text-warning" ng-bind="ctrl.statusMessage"></h3>
     <div class="well well-sm">
     <form>
       <strong>Min edge weight:</strong>
-      <label>
-        <input type="checkbox" ng-model="ctrl.includeNegative"
-               ng-disabled="!ctrl.includePositive"
-               ng-change="ctrl.renderNetwork()">
-        Include negative weight
-      </label>
+      <input type="radio" name="edgeSign" ng-model="ctrl.edgeSign"
+             ng-change="ctrl.renderNetwork()"
+             value="negOnly">Negative only
```

Comment:
I noticed on the demo page that there is no space between the radio button and its label and they appear to "collide" with each other on the page. The code here is a little different from what is shown in the AngularJS documentation:

> https://code.angularjs.org/1.5.11/docs/api/ng/input/input[radio]

In the Angular sample code there is a newline between the `<input>` tag and the label. There is also a `<label>` tag wrapping the `<input>`. Probably a space or newline would be sufficient here but if that doesn't help, try using the `<label>` wrapper as well (as you had before with the checkboxes).